waves-integration/src/main/scala/com/wavesplatform/dex/domain/account/AddressOrAlias.scala

```diff
@@ -11,8 +11,10 @@ trait AddressOrAlias {
 
   override def toString: String = stringRepr
 
+  def canEqual(other: Any): Boolean = other.isInstanceOf[AddressOrAlias]
+
   override def equals(obj: Any): Boolean = obj match {
-    case a: AddressOrAlias => bytes == a.bytes
+    case a: AddressOrAlias => a.canEqual(this) && bytes == a.bytes && stringRepr == a.stringRepr
```

Comment:
Actually, we don't need to compare `stringRepr` because it is a representation of `bytes`

Reply:
fixed